packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -904,7 +907,9 @@ export class SideTabBar extends ScrollableTabBar {
             } else {
                 rd = { title, current, zIndex };
             }
-            content[i] = renderer.renderTab(rd, true);
+            // Based on how renderTabs() is called, assuming renderData will be undefined when
+            // invoked for this.hiddenContentNode
```

Comment:
minor: I believe you meant `assume` and not `assuming` here:

```suggestion
            // Based on how renderTabs() is called, assume renderData will be undefined when invoked for this.hiddenContentNode.
```